crates/bitwarden/src/crypto/enc_string.rs

```diff
@@ -331,6 +332,20 @@ impl serde::Serialize for EncString {
 }
 
 impl EncString {
+    pub(crate) fn encrypt_aes256_hmac(
+        data_dec: &[u8],
+        mac_key: GenericArray<u8, U32>,
+        key: GenericArray<u8, U32>,
+    ) -> Result<EncString> {
+        let enc = super::encrypt_aes256_hmac(data_dec, mac_key, key)?;
+
+        Ok(EncString::AesCbc256_HmacSha256_B64 {
+            iv: enc.0,
+            mac: enc.1,
+            data: enc.2,
+        })
```

Comment:
Small style suggestion:
```suggestion
        let (iv, mac, data) = super::encrypt_aes256_hmac(data_dec, mac_key, key)?;
        Ok(EncString::AesCbc256_HmacSha256_B64 { iv, mac, data })
```